resumablejs/resumable.d.ts

```diff
@@ -0,0 +1,374 @@
+// Type definitions for Resumable.js v1.0.2
```

Comment:
filename should match the package name. so it should be `resumablejs/resumablejs.d.ts`
see http://definitelytyped.org/guides/contributing.html#naming-the-file